ee/tabby-ui/lib/utils.ts

```diff
@@ -78,3 +78,13 @@ export const delay = (ms: number) => {
     setTimeout(() => resolve(null), ms)
   })
 }
+
+export const convertToDisplayName = (lan: string): string => {
+  const mapping: Record<string, string> = {
+    csharp: 'C#',
+    cpp: 'C++',
+    javascript: 'JavaScript',
+    typescript: 'TypeScript'
+  }
+  return mapping[lan.toLocaleLowerCase()] || lan
+}
```

Comment:
could you:
1. rename `@/lib/filename2prism` to `language-utils`
2. export `filename2prism` and `toProgrammingLanguageDisplayName` from this utils.